app/[locale]/(dashboard)/sensors/[id]/page.jsx

```diff
@@ -0,0 +1,26 @@
+import { getServerSession } from 'next-auth/next'
+import { authOptions } from '@/lib/auth'
+import { Page } from '@/components/Page/Page'
+import { PageHeader } from '@/components/Page/PageHeader'
+import { IconButton } from '@/components/ui/IconButton/IconButton'
+import { Left } from '@/components/ui/Icons'
+import { Link } from '@/components/ui/Link'
+
+async function Home({ params }) {
+  const session = await getServerSession(authOptions)
+
+  if (!session) {
+    redirect('/auth/login')
+  }
+
+  return (
+    <Page>
+      <IconButton icon={Left} variant="secondary">
+        <Link href={'/sensors'}></Link>
+      </IconButton>
```

Comment:
We can add the back button as a part of the PageHeader

Reply:
I did that. Doing the PR today!

Reply:
Cool but merge this PR before doing new change